src/App.jsx

```diff
@@ -1,107 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
-
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+      || null;
+}
+
+function getCommentById(userId) {
+  return commentsFromServer.filter(comment => comment.postId === userId)
+      || null;
```

Comment:
I guess there is no need to write `null` here because if there is no comment will be found then we have just an empty array which is always be like `true` if we transform it into boolean value